chain/chain/src/delay_detector.rs

```diff
@@ -0,0 +1,25 @@
+use log::info;
```

Comment:
why is this file part of the PR? Delay detector is already merged in #3126

Reply:
I was using an older version, and this remained after rebasing.